.github/workflows/build.yml

```diff
@@ -0,0 +1,325 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  azCliVersion: 2.6.0
+  ref_javaee: 6addd99d8bc3f472e040f11c053a37e1ac370229
+  offerName: azure.websphere-traditional.image
+  userEmail: ${{ secrets.USER_EMAIL }}
```

Comment:
Whose email is this and how is this variable used?

Reply:
It does not appear to be used.  I will delete it.